test/externalissuer/external_issuer_test.go

```diff
@@ -89,16 +91,21 @@ func verifyAppWorksBeforeCertRotation(t *testing.T) {
 }
 
 func verifyRotateExternalCerts(t *testing.T) {
-	// change issuer secret
-	secretResource, err := testutil.ReadFile("testdata/issuer_secret_2.yaml")
+	// We rotate the certificates here by simply grabbing
+	// the key and cert values from the temporary secret we have
+	// created
+	secretWithUpdatedData, err := TestHelper.KubernetesHelper.GetSecret(TestHelper.GetLinkerdNamespace(), k8s.IdentityIssuerSecretName+"-new")
+
```

Comment:
Nit: Mind removing the extra line break, to be consistent with everywhere else? Most Go code I have seen groups the `if err != nil` immediately after the `err` declaration.